src/cpu_feature/arm.rs

```diff
@@ -0,0 +1,281 @@
+use std::collections::HashSet;
+use std::string::{String, ToString};
+
+use super::libc::c_ulong;
```

Comment:
If you're not worried about maintaining the platform/bitwidth mapping, sounds good to me!